### PR TITLE
Rise: Update bidder info yaml

### PR DESCRIPTION
**Pull request by zkosanovic**

### CODE REVIEW

File: static/bidder-info/rise.yaml [hunk 1]
```diff
@@ -8,6 +11,7 @@ capabilities:
     mediaTypes:
       - banner
       - video
+      - native
```
Comment (SyntaxNode) [inline static/bidder-info/rise.yaml:14]: Adding a new media type requires accompanying json tests with requests for that new media type.

**Review by SyntaxNode — COMMENTED** [review]
As you're updating the bidder info, does Rise support OpenRTB 2.6? That's another potential setting to declare in the bidder info.

File: static/bidder-info/rise.yaml (hunk 1 above)
Comment (zkosanovic) [reply static/bidder-info/rise.yaml:14]: Added a native JSON test

**Comment by zkosanovic** [comment]
@SyntaxNode as of right now, we don't support OpenRTB 2.6

File: adapters/rise/risetest/exemplary/simple-native.json [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                  "impid": "test-imp-id",
+                  "price": 0.5,
+                  "adid": "29681110",
+                  "adm": "some-test-ad",
+                  "adomain": ["yahoo.com"],
+                  "cid": "958",
+                  "crid": "29681110",
+                  "h": 250,
+                  "w": 300,
+                  "mtype": 1
```
Comment (SyntaxNode) [inline adapters/rise/risetest/exemplary/simple-native.json:65]: The `mtype` should be `4` for native.

Comment (zkosanovic) [reply adapters/rise/risetest/exemplary/simple-native.json]: Thanks, fixed

**Review by SyntaxNode — APPROVED** [review]

**Review by bsardo — COMMENTED** [review]
Please open up a docs PR to capture these changes and provide the link here.

**Review by bsardo — APPROVED** [review]

**Pull request merged**